Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND="noninteractive" 
+
+RUN apt-get update -qq && apt-get install -y \
+    git \
+    cmake \
+    g++ \
+    pkg-config \
+    libssl-dev \
+    curl \
+    llvm \
+    clang \
+    libpq-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY ./rust-toolchain /tmp/rust-toolchain
+
+ENV RUSTUP_HOME=/usr/local/rustup \
+    CARGO_HOME=/usr/local/cargo \
+    PATH=/usr/local/cargo/bin:$PATH
+
+RUN curl https://sh.rustup.rs -sSf | \
+    sh -s -- -y --no-modify-path --default-toolchain "$(cat /tmp/rust-toolchain)"
+
+RUN cargo install diesel_cli --no-default-features --features "postgres"
+COPY . .
+RUN cargo build --release -p indexer-explorer
```

Comment:
I would create a second stage extracting the binary itself, and the nearcore home folder to greatly slim down the image

Reply:
Did some experiments with it, it's not as easy, there are some runtime libs it requires, I've tried it using musl but it wasn't working properly

Reply:
It needs only `libpq-dev` if I'm not mistaken